tests/console/yast2_cmdline.pm

```diff
@@ -40,8 +42,25 @@ sub run_yast_cli_test {
 }
 
 sub run {
+    my $cmd;
+
     select_console 'root-console';
 
+    if (is_sle) {
+        if (is_sle('15+') and get_var('REPO_SLE15_MODULE_BASESYSTEM_SOURCE')) {
+            $cmd = "ar -f " . "$utils::OPENQA_FTP_URL/" . get_var('REPO_SLE15_MODULE_BASESYSTEM_SOURCE') . " repo-source";
+        }
+        elsif (is_sle('<15') and get_var('REPO_SLES_SOURCE')) {
+            $cmd = "ar -f " . "$utils::OPENQA_FTP_URL/" . get_var('REPO_SLES_SOURCE') . " repo-source";
+        }
+    }
+    else {
+        die 'Repo syncing issue on openSUSE';
+    }
+
+    zypper_call($cmd) if defined $cmd;
+    zypper_call("ref");
```

Comment:
Why do you still have this part when you stated that `-f` ensures the repos are refreshed on first use?